doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -1,84 +1,92 @@
-==========================
-BlueStore Config Reference
-==========================
+==================================
+ BlueStore Configuration Reference 
+==================================
 
 Devices
 =======
 
-BlueStore manages either one, two, or (in certain cases) three storage
-devices.
+BlueStore manages either one, two, or in certain cases three storage devices.
 
-In the simplest case, BlueStore consumes a single (primary) storage device.
-The storage device is normally used as a whole, occupying the full device that
-is managed directly by BlueStore. This *primary device* is normally identified
-by a ``block`` symlink in the data directory.
+In the simplest case, BlueStore consumes a single storage device. The device is
+known as the *primary device*. Any device used by BlueStore will be used
+entirely by BlueStore and nothing other than BlueStore. The primary device is
```

Comment:
I'm a bit uncertain about this rewording, I think it diverges from the intent of the original and in some situations is misleading.  I propose:

>BlueStore manages either one, two, or in certain cases three storage devices.  These _devices_ are in the Linux / Unix sense of something that appears under `/dev` or `/devices`, and may be an entire storage drive, a partition of a storage drive, or a logical volume.  BlueStore does not create or mount a conventional filesystem on devices that it uses, it read and writes to them directly in a "raw" fashion.

>In the simplest case, BlueStore consumes all of a single storage device, which we call the `primary device`.  The primary device is identified by the ``block`` symlink in the metadata directory.

Reply:
Largely accepted with very minor changes.